cypress/e2e/functional/researcher_anonymous_classes.spec.js

```diff
@@ -0,0 +1,149 @@
+import * as c from '../../support/constants.js'
+import * as researchClassesHelper from "../../support/helpers/researcherClassesHelper.js"
+import {ResearcherClassesElements as researchClasses} from "../../support/elements/researcher_anonymous_classes_elements";
+
+context("Researcher classes side bar, filters, resutls table", () => {
```

Comment:
Misspelling: "resutls" should be "results".

Reply:
Fixed